app/domain/authentication/resource_restrictions/extract_resource_restrictions.rb

```diff
@@ -5,22 +5,25 @@ module ResourceRestrictions
 
     ExtractResourceRestrictions = CommandClass.new(
       dependencies: {
-        resource_restrictions_class: ResourceRestrictions::ResourceRestrictions,
+        resource_restrictions_class: Authentication::ResourceRestrictions::ResourceRestrictions,
+        get_restriction_from_annotation: Authentication::ResourceRestrictions::GetRestrictionFromAnnotation.new,
+        fetch_resource_annotations: Authentication::ResourceRestrictions::FetchResourceAnnotations.new,
         role_class: ::Role,
         resource_class: ::Resource,
-        logger: Rails.logger
+        logger: Rails.logger,
+        ignore_empty_annotations: true
       },
       inputs: %i[authenticator_name service_id role_name account]
     ) do
       def call
-        @logger.debug(
+        @logger.info(
```

Comment:
why do we need this in info?

Reply:
Approved by @hilagross also raised that it is a behavior change in all other authenticators